.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+on:
```

Comment:
This doesn't seem to be running all the different combinations of tests.  I thought we ran against postgres, mysql and sqlite.  Also, I thought we tested different templating options using docker compose.

Reply:
I'll look into expanding the combination of tests more. I'm not very familiar with GHA and I'm just trying to make it work to start.

Reply:
And FYI I still have 10 failing tests to fix, so I won't merge this until at least everything is green on my repo